server/services/participant-status/handle-transition.ts

```diff
@@ -54,16 +54,39 @@ const handleArchivedTransition = async ({
 
   // Withdraw participant from program
   if (!isRosComplete) {
-    await withdrawParticipant(participant[0]);
+    await withdrawParticipant(participant);
   }
 
   return {};
 };
 
-const handleHiredTransition = async ({ tx, existingCurrentStatus, site, participantId }) => {
+// Handles the logic when a participant gets hired into a site
```

Comment:
Make this a JSDoc comment so it is visible in IDEs. Also add argument typing if possible.